open-api.yaml

```diff
@@ -0,0 +1,1614 @@
+      items:
+        oneOf:
+          - type: string
+            pattern: '^[^:]+:[^:]+$'
+          - type: array
+            items:
+              type: string
+              pattern: '^[^:]+:[^:]+$'
+      example:
+        - 'director:Mati Diop'
+        - - 'genres:Comedy'
+          - 'genres:Romance'
+      title: filter
+    searchResponse:
+      type: object
+      additionalProperties: false
+      properties:
+        hits:
+          type: array
+          description: Results of the query.
+          items:
+            $ref: '#/components/schemas/hit'
+        offset:
+          type: integer
+          description: Number of documents skipped.
+        limit:
+          type: integer
+          description: Number of documents to take.
+        nbHits:
+          type: integer
+          description: Total number of matches.
+        exhaustiveNbHits:
+          type: boolean
+          description: Whether `nbHits` is exhaustive.
+        facetsDistribution:
+          type: object
+          additionalProperties:
+            type: object
+            additionalProperties:
+              type: integer
+          description: |
+            [Distribution of the given facets](/reference/features/search_parameters.html#the-facets-distribution).
+          example:
+            genres:
+              action: 273
+              animation: 118
+              adventure: 132
+              fantasy: 67
+              comedy: 475
+              mystery: 70
+              thriller: 217
+        exhaustiveFacetsCount:
+          type: boolean
+          description: Whether `facetsDistribution` is exhaustive.
+        processingTimeMs:
+          type: integer
+          description: Processing time of the query.
+        query:
+          type: string
+          description: Query originating the response.
+          example: ninja
+      required:
+        - hits
+        - offset
+        - limit
+        - nbHits
+        - exhaustiveNbHits
+        - processingTimeMs
+        - query
+      title: ''
+    update:
+      type: object
+      description: |
+        MeiliSearch is an asynchronous API. It means that the API does not behave as you would typically expect when handling the request's responses.
+
+        Some operations are put in a queue and will be executed in turn (asynchronously). In this case, the server response contains the identifier to track the execution of the operation.
+
+        [Learn More](/learn/advanced/asynchronous_updates.html).
+      additionalProperties: false
+      properties:
+        status:
+          description: The state of the operation.
+          enum:
+            - enqueued
+            - processed
+            - failed
+        updateId:
+          type: integer
+          description: The id of the update.
+        type:
+          type: object
+          additionalProperties: false
+          required:
+            - name
+            - number
```

Comment:
Not the case for `number` meilisearch/MeiliSearch/issues/1517

Reply:
Nice spot ;) Sometimes it can be `settings`